Core/src/org/sleuthkit/autopsy/datamodel/AbstractAbstractFileNode.java

```diff
@@ -457,4 +582,36 @@ protected static String getHashSetHitsCsvList(AbstractFile file) {
             return "";
         }
     }
+
+    /**
+     * Fill map with AbstractFile properties
+     *
+     * @param map     map with preserved ordering, where property names/values
+     *                are put
+     * @param content The content to get properties for.
+     */
+    @Deprecated
```

Comment:
Since we are still using this method, please remove the deprecation and make a technical debt story about removing this method in the future. You can reference the story in the code in a comment using the form "TODO JIRA-XXXX:"